src/handbook/peopleops/expenses.md

```diff
@@ -4,32 +4,22 @@ navTitle: Expenses
 
 ## Expenses
 
-The company will cover any reasonable work-related expense, including ensuring you
-have the equipment (laptop/monitor/accessories) you need to do the job. As an
-all-remote company, we meet via Zoom. As such, we will cover the cost of a good
-webcam/microphone/lighting arrangement.
+FlowForge will reimburse you for any work-related expenses that are considered reasonable. This includes providing you with the necessary equipment, such as a laptop, monitor, and accessories, to effectively carry out your job responsibilities. As an all-remote company, our meetings are conducted using either Zoom or Slack. Therefore, we will also cover the expenses associated with obtaining a high-quality webcam, microphone, and lighting setup.
 
-The general rule is: treat it like your own money - seek good value and use 
-common sense. If you do require any equipment, talk to your manager to discuss options.
+The general principle to follow is to treat work-related expenses as if they were your own money. It is important to seek good value for your purchases and use common sense when making decisions. If you find yourself in need of any equipment, it is advisable to have a discussion with your manager to explore the available options.
 
-There are two ways of spending company money - either using a company credit card,
-or through reimbursement.
+There are two methods available for using company funds: utilizing a company credit card or seeking reimbursement. If employees opt to use Brex for purchasing any company-related items or subscriptions, it is essential to upload the receipts and include a memo within three business days. For all reimbursement requests, please ensure to submit the required documentation and proofs through Deel.
```

Comment:
Subscriptions need approval, as vendors require approval. @knolleary is working on a list of currently approved vendors. We should link that here.